tooling/Microsoft.VisualStudio.BlazorExtension/Properties/AssemblyInfo.cs

```diff
@@ -8,13 +8,6 @@
 //
 // The versions here need to match what the build is producing. If you change the version numbers
 // for the Blazor assemblies, this needs to change as well.
-[assembly: ProvideBindingRedirection(
-    AssemblyName = "Microsoft.AspNetCore.Blazor.AngleSharp",
-    GenerateCodeBase = true,
-    PublicKeyToken = "",
-    OldVersionLowerBound = "0.0.0.0",
-    OldVersionUpperBound = "0.9.9.0",
-    NewVersion = "0.9.9.0")]
```

Comment:
This will still be needed. If someone else specifies a different version of anglesharp in VS, we want ours to win if it's the newest.